diff --git a/node_modules/docusaurus/lib/publish-gh-pages.js b/node_modules/docusaurus/lib/publish-gh-pages.js
index 221f1a9..9070828 100755
--- a/node_modules/docusaurus/lib/publish-gh-pages.js
+++ b/node_modules/docusaurus/lib/publish-gh-pages.js
@@ -114,6 +114,14 @@ if (
   shell.exit(1);
 }
 
+if (PROJECT_NAME.indexOf("polkadot") !== -1) {
+  shell.exec("yarn polkadot:inject");
+  shell.exec("yarn polkadot:remove");
+} else {
+  shell.exec("yarn kusama:inject");
+  shell.exec("yarn kusama:remove");
+}
+
 shell.cd(`${PROJECT_NAME}-${DEPLOYMENT_BRANCH}`);
 
 // If the default branch is the one we're deploying to, then we'll fail to create it.
